scripts/install_bitcoind.sh

```diff
@@ -2,9 +2,9 @@
 
 set -ev
 
-BITCOIND_VERSION=${BITCOIN_VERSION:-0.20.1}
+BITCOIND_VERSION=${BITCOIN_VERSION:-22.0}
 
-docker pull ruimarinho/bitcoin-core:$BITCOIND_VERSION
-CONTAINER_ID=$(docker create ruimarinho/bitcoin-core:$BITCOIND_VERSION)
+docker pull lightninglabs/bitcoin-core:$BITCOIND_VERSION
```

Comment:
Why not use ghcr? Like you had mentioned it doesn't have throttling of requests. 

Also the down side (which isn't IMO) to ghcr all requests have to authenticated. By that it means the user needs to have a GitHub ID.

Reply:
We want to switch over, but everything else (about 5 other repos) still uses Docker Hub, so we'll add this for now, then switch everything over in one swoop.